tests/ga/test_exthandlers.py

```diff
@@ -76,6 +76,57 @@ def test_parse_extension_status01(self):
         self.assertEqual(0, ext_status.sequenceNumber)
         self.assertEqual(0, len(ext_status.substatusList))
 
+    def test_parse_ext_status_should_parse_missing_substatus_as_empty(self):
+        status = '''[{
+            "status": {
+              "status": "success",
+              "formattedMessage": {
+                "lang": "en-US",
+                "message": "Command is finished."
+              },
+              "operation": "Enable",
```

Comment:
The status in `test_parse_extension_status00` also doesn't have the substatus field. in Line 28. Just curious - could this new `test_parse_ext_status_should_parse_missing_substatus_as_empty` test be merged into that test ?

Its a matter of adding the following two lines.
```
self.assertTrue(isinstance(extension_status.substatusList, list), 'substatus was not parsed correctly')
self.assertEqual(0, len(extension_status.substatusList))
```

Reply:
The intention of test_parse_extension_status00 seems to be to verify the basic parsing of a successful status.

While writing test_parse_ext_status_should_parse_null_substatus_as_empty, I decided the behavior for a missing (not specified) substatus was also important enough to call it out in its own test.